Graphene-1.0.gir

```diff
@@ -3200,23 +3194,6 @@ or graphene_point_init_from_point(), e.g.:
           </parameter>
         </parameters>
       </method>
-      <method name="distance_squared" c:identifier="graphene_point_distance_squared" version="1.12">
```

Comment:
Why is this removed?

Reply:
It was just added in https://github.com/ebassi/graphene/commit/362c9d8bee291d08ef46dce3027aac7e56aeff07 . Did the gir file get downgraded in the GNOME SDK?